rs/bitcoin/ckbtc/minter/BUILD.bazel

```diff
@@ -234,3 +250,50 @@ genrule(
 """,
     visibility = ["//visibility:private"],
 )
+
+rust_canbench(
+    name = "ckbtc_minter_canbench",
+    srcs = ["src/main.rs"],
+    compile_data = [":ckbtc_minter.did"],
+    data = ["mainnet_events.mem.gz"],
+    env = {
+        "CANBENCH_STABLE_MEMORY_FILE": "$(location mainnet_events.mem.gz)",
+        # the following hex arg was produced by command:
+        #
+        #   didc encode -d ./ckbtc_minter.did -t '(MinterArg)' '(
+        #     variant {
+        #       Init = record {
+        #         ecdsa_key_name = "master_ecdsa_public_key";
+        #         mode = variant { GeneralAvailability };
+        #         retrieve_btc_min_amount = 100_000 : nat64;
+        #         ledger_id = principal "lxzze-o7777-77777-aaaaa-cai";
+        #         max_time_in_queue_nanos = 100 : nat64;
+        #         btc_network = variant { Mainnet };
+        #         check_fee = opt (100 : nat64);
+        #         btc_checker_principal = opt principal "lz3um-vp777-77777-aaaba-cai";
+        #         min_confirmations = opt (6 : nat32);
+        #       }
+        #     }
+        #   )'
+        "CANBENCH_INIT_ARGS_HEX": "4449444c0b6b02fcb88b840301b0ced18403096e026c08f5d19a6b03e3a683c30404b6b8ba8405078ebdcfc608078fc0fec00a07b682e7c90b03f18c92d50b08cdb2eee10c076e686e056b0496bab03c068b98eaed020682b8f0da047f8390999e057f6d686e786e796c0bf5d19a6b03b6a9f6dc0271e3a683c30405b6b8ba840578f1f7fcf706688ebdcfc6087880b3d0ed090a8fc0fec00a07b682e7c90b03f18c92d50b08cdb2eee10c076b03a4d3bb457f8687acf9047fabdeaed9087f01000100176d61737465725f65636473615f7075626c69635f6b657903a086010000000000010affffffffffe00000010164000000000000000001640000000000000001010affffffffffe000020101010600000000",
+    },
+    proc_macro_deps = LIB_PROC_MACRO_DEPS,
+    results_file = "canbench/results.yml",
+    rustc_env = {
+        "CKBTC_MINTER_DID_PATH": "$(execpath :ckbtc_minter.did)",
```

Comment:
would it make sense to use `export_service!()` instead of including the external did file?

Reply:
The did file was actually hand written (in order to have documentation, etc.). We only use `export_service` to check compatibility but not for generating the did file.